ci/playbooks/edpm/update.yml

```diff
@@ -31,5 +31,6 @@
           -e "{{   extra_var }}"
           {%-   endfor %}
           {%- endif %}
+          -e update_playbook_run=true
```

Comment:
this seems fine but wondering if we can use the existing cifmw_extras ? going to approve so definitely not blocking on that just a thought

Reply:
cifmw_extras are defined in deployment run too. In update CI run we first start deployment and next update playbook and this flag differentiates this step from deployment.

Reply:
it is used in: https://github.com/openstack-k8s-operators/ci-framework/pull/1229